src/components/molecules/ResultsDatePicker/ResultDatePicker.tsx

```diff
@@ -27,28 +27,25 @@ const datePickerStyles = {
 };
 
 const ResultDatePicker = () => {
-  const [toggleGetTest, setToggleGetTest] = useState<boolean>(false);
   const [selectedDate, setSelectedDate] = useState<Moment>();
   const tests: any = React.useContext(TestsContext);
 
   const handleDatePicker = (value: any, dateString: any) => {
     tests.setSelectedTest({ id: null, testName: null });
-    setSelectedDate(value);
-    tests.filters.dateFilter = dateString;
 
-    tests.setFilters(tests.filters);
+    setSelectedDate(value);
+    tests.setFilterByDate(dateString);
 
   };
 
   const getTodayTests = React.useCallback(() => {
     tests.setSelectedTest({ id: null, testName: null });
     let currentDate = moment();
     setSelectedDate(currentDate);
+    tests.setFilterByDate(currentDate.format('MM-DD-YYYY'));
```

Comment:
dateformat is yyyy-mm-dd - see https://github.com/kubeshop/testkube/blob/main/api/v1/testkube.yaml#L764 - @nicufk please confirm